lesson2/main.js

```diff
@@ -0,0 +1,17 @@
+// create list property
+const li = document.createElement("li");
+
+// create anchor property
+const a = document.createElement("a");
+a.href = "1.html";
+a.textContent = "これです";
+
+// create img property
+const img = document.createElement("img");
+img.src = "bookmark.png";
+img.alt = "ブックマーク";
+
+// add them in ul property
+const ul = document.getElementById("js-lists");
+
```

Comment:
【IMO】
It will be easier to read the code if you write it separately for each variable definition, instructions for elements, and DOM operations.

Reply:
You're right.I changed, thank you!